src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/LSPRequestInvoker.cs

```diff
@@ -44,13 +67,36 @@ internal abstract class LSPRequestInvoker
             CancellationToken cancellationToken)
             where TIn : notnull;
 
+        /// <summary>
+        /// Reinvoke the request on the given server.
+        /// </summary>
+        /// <typeparam name="TIn"></typeparam>
+        /// <typeparam name="TOut"></typeparam>
+        /// <param name="method"></param>
+        /// <param name="contentType"></param>
+        /// <param name="parameters"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        /// <remarks>When operating on a document the <see cref="ITextBuffer"/> overload should be used, since it guarentees ordering.</remarks>
```

Comment:
nit:

```suggestion
        /// <remarks>When operating on a document the <see cref="ITextBuffer"/> overload should be used, since it guarantees ordering.</remarks>
```

(same for other doc comments in this file)